### PR TITLE
Update MidnightCommander to 4.8.24

### PR DESCRIPTION
**Pull request by Mno-hime**
**Changes**: https://github.com/MidnightCommander/mc/blob/4.8.24/doc/NEWS

### CODE REVIEW

File: components/sysutils/mc/Makefile [hunk 1]
```diff
@@ -37,13 +38,11 @@ COMPONENT_PROJECT_URL=	http://www.midnight-commander.org/
 COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
 COMPONENT_ARCHIVE_HASH=	\
-	sha256:ee7868d7ba0498cf2cccefe107d7efee7f2571098806bba2aed5a159db801318
+	sha256:859f1cc070450bf6eb4d319ffcb6a5ac29deb0ac0d81559fb2e71242b1176d46
 COMPONENT_ARCHIVE_URL=	http://ftp.midnight-commander.org/$(COMPONENT_ARCHIVE)
 COMPONENT_LICENSE=	GPLv3
```
Comment (AndWac) [inline components/sysutils/mc/Makefile:44]: Are there tests now or did you forget to add
TEST_TARGET=            $(NO_TESTS)
here?

Comment (Mno-hime) [reply components/sysutils/mc/Makefile:44]: Yes, there are tests, we just didn't run them before.

**Comment by pyhalov** [comment]
Is it expected:
```
FAIL: path_cmp
FAIL: path_len
FAIL: path_manipulations
FAIL: path_serialize
``` 
?

**Comment by pyhalov** [comment]
OK, found this bug report - https://midnight-commander.org/ticket/3972
Can you add expected test results?

**Comment by pyhalov** [comment]
@Mno-hime Can you add expected test results?

**Comment by Mno-hime** [comment]
> @Mno-hime Can you add expected test results?

@pyhalov Test record added.

**Pull request merged**